fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Variable.java

```diff
@@ -47,6 +47,11 @@ public String getName() {
         return name;
     }
 
+    @Override
+    protected String getExpressionName() {
```

Comment:
Seems same as Expression.getExpressionName()?

Reply:
yeah, this is same to `Expression.getExpressionName()`, it can be removed. I will fix it.